daemon/daemon.go

```diff
@@ -239,25 +239,25 @@ func (daemon *Daemon) restore() error {
 				log.WithError(err).Error("failed to load container")
 				return
 			}
-			// Ignore the container if it does not support the current driver being used by the graph
-			if (c.Driver == "" && daemon.graphDriver == "aufs") || c.Driver == daemon.graphDriver {
-				rwlayer, err := daemon.imageService.GetLayerByID(c.ID)
-				if err != nil {
-					log.WithError(err).Error("failed to load container mount")
-					return
-				}
-				c.RWLayer = rwlayer
-				log.WithFields(logrus.Fields{
-					"running": c.IsRunning(),
-					"paused":  c.IsPaused(),
-				}).Debug("loaded container")
-
-				mapLock.Lock()
-				containers[c.ID] = c
-				mapLock.Unlock()
-			} else {
+			if c.Driver != daemon.graphDriver {
+				// Ignore the container if it does not support the current driver being used by the graph
 				log.Debugf("cannot load container because it was created with another storage driver")
```

Comment:
Shouldn't this message include one or both of `c.Driver` / `daemon.graphDriver`? :eyes:

(I realize you only moved the existing message, but it's what jumps out at me now that the logic is so much clearer :joy:)

Reply:
Good suggestion; yes, might as well improve that while we're changing this; I'll have a look later

Reply:
updated the log message (decided to only include the container's driver, to make it not _too_ verbose)